jib-maven-plugin/src/main/java/com/google/cloud/tools/jib/maven/ExtraDirectory.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2020 Google LLC.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.google.cloud.tools.jib.maven;
+
+import java.io.File;
+import java.nio.file.Path;
+import java.util.Objects;
+import org.apache.maven.plugins.annotations.Parameter;
+
+/** A bean that configures the source and destination of an extra directory. */
+public class ExtraDirectory {
```

Comment:
Can we name this `ExtraDirectoryParameters` and move it to `JibPluginConfiguration` for consistency?